lighthouse-core/audits/dobetterweb/has-inspector-issues.js

```diff
@@ -0,0 +1,246 @@
+  mixedContentMessage: 'Some resources like images, stylesheets or scripts ' +
+    'are being accessed over an insecure `HTTP` connection',
+  /** Message shown in a data table when the item is a BlockedByResponse issue. This is when a resource is blocked due to not being allowed by a Cross-Origin Embedder Policy. */
+  coepResourceBlockedMessage: 'A resource was blocked due to not being ' +
+    'allowed by a `Cross-Origin Embedder Policy`',
+  /** Message shown in a data table when the item is a BlockedByResponse issue. This is when a frame is blocked due to not being allowed by a Cross-Origin Embedder Policy. */
+  coepFrameBlockedMessage: 'A frame was blocked due to not being ' +
+    'allowed by a `Cross-Origin Embedder Policy`',
+  /** Message shown in a data table when the item is a BlockedByResponse issue. This is when navigation to a document with a Cross-Origin Opener Policy is blocked. */
+  coopIframeBlockedMessage: 'An iframe navigation to a document with ' +
+    'a `Cross-Origin Opener Policy` was blocked',
+  /** Message shown in a data table when the item is a HeavyAds issue where an ad uses more than 4 megabytes of network bandwith. */
+  heavyAdsNetworkLimitMessage: 'The page contains ads that use ' +
+    'more than 4 megabytes of network bandwidth',
+  /** Message shown in a data table when the item is a HeavyAds issue where an ad has used the main thread for more than 60 seconds in total. */
+  heavyAdsCPUTotalLimitMessage: 'The page contains ads that use the ' +
+    'main thread for more than 60 seconds in total',
+  /** Message shown in a data table when the item is a HeavyAds issue where an ad has used the main thread for more than 15 seconds in any 30 second window. */
+  heavyAdsCPUPeakLimitMessage: 'The page contains ads that use the ' +
+    'main thread for more than 15 seconds in a 30 second window',
+  /** Message shown in a data table when the item is a ContentSecurityPolicy issue where resources are blocked due to not being in the Content Security Policy header. */
+  cspUrlViolationMessage: 'The `Content Security Policy` of the page blocks some resources because ' +
+    'their origin is not included in the content security policy header',
+  /** Message shown in a data table when the item is a ContentSecurityPolicy issue where the Content Security Policy blocks inline execution of scripts and stylesheets. */
+  cspInlineViolationMessage: 'The `Content Security Policy` of the page blocks ' +
+    'inline execution of scripts and stylesheets',
+  /** Message shown in a data table when the item is a ContentSecurityPolicy issue where the Content Security Policy blocks the use of the `eval` function in Javascript. */
+  cspEvalViolationMessage: 'The `Content Security Policy` of the site blocks ' +
+    'the use of `eval` in JavaScript',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+/** @type {Record<string, string>} */
```

Comment:
These are all going to be `LH.IcuMessage` after #10630 (need to rebase/merge, of course)

```suggestion
/** @type {Record<string, LH.IcuMessage>} */
```